docs/installation/uninstall.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Uninstall Opni
+slug: /installation/uninstall
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Before uninstalling all capabilities should be removed from all clusters.  This can be done by clicking an installed capability in the clusters screen and choosing to remove the cabability.
```

Comment:
```suggestion
Before uninstalling, all capabilities should be removed from all clusters.  This can be done by clicking an installed capability in the clusters screen and choosing to remove the capability.
```